modules/KoreBuild.Tasks/Utilities/VsWhere.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+
+namespace KoreBuild.Tasks.Utilities
+{
+    internal class VsWhere
+    {
+        public static VsInstallation FindLatestInstallation(bool includePrerelease, TaskLoggingHelper log)
+        {
+            var args = new List<string>
+            {
+                "-latest",
+            };
+            if (includePrerelease)
+            {
+                args.Add("-prerelease");
+            }
+
+            return GetInstallations(args, log).FirstOrDefault();
+        }
+
+        public static VsInstallation FindMostCompatibleInstallation(KoreBuildSettings.VisualStudioToolset toolset, TaskLoggingHelper log)
```

Comment:
The method seems to be finding the first compatible Installation. Rename to "FindFirstCompatibleInstallation"?

Reply:
'FindLatestCompatibleInstallation' would be better. It finds the newest VS that meets the given requirements. "first" is a little ambiguous and implies there is some kind of sorting on name or version.